docs/proposal-0003-moving-configs-into-metric-hub.md

```diff
@@ -0,0 +1,61 @@
+# Moving configs to metric-hub
+
+* Date: 2023-01-16
+
+## Context and Problem Statement
+
+Configurations for jetstream currently live in the [jetstream-config](https://github.com/mozilla/jetstream-config) repo. The config files reference metrics that are part of the [metric-hub](https://github.com/mozilla/metric-hub) repo. The separation of configs of both repositories has a few consequences:
+
+* If a new jetstream config introduces a new metric that should be part of metric-hub, then multiple pull-requests that depend on each other need to be opened. The PR in jetstream-config that references the new metric will cause the CI to fail until the metric is available in metric-hub.
```

Comment:
A workaround for this would be to just define the metric in the experiment config until it is approved in `metric-hub`. This seems like a reasonable flow to me anyway (almost like a beta version for a specific experiment, and then it gets pushed to metric-hub when approved for broader release).

Reply:
Yeah, I think that is the current workaround. Though it does require making three sets of changes vs one (adding the config, adding the metric to metric-hub and updating the config after metrics are available).